public/js/createSrcSet.js

```diff
@@ -0,0 +1,15 @@
+
+fetch("../sourceSets.json").then(response => { return response.json();})
```

Comment:
In the deploy preview, this is failing to grab the file. That URL resolves to:

https://deploy-preview-226--refcode.netlify.app/sourceSets.json

which is a 404.

Reply:
Ignore the above, after running the script, that file does exist (see this deploy preview: https://611a67a88af9b53f1184ac09--refcode.netlify.app/sourceSets.json)